src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -107,6 +110,9 @@ export interface NetEmitterOptions {
     "clear-output-folder"?: boolean;
     "save-inputs"?: boolean;
     "model-namespace"?: boolean;
+    "generate-protocol-methods"?: boolean;
+    "generate-convenience-methods"?: boolean;
+    "package-name"?: string;
 }
 
 const defaultOptions = {
```

Comment:
Need to set the default value for 'generate-protocol-methods` and `generate-convenience-methods`